mozfun/glean/percentile/metadata.yaml

```diff
@@ -0,0 +1,2 @@
+description: 'Glean Percentile'
```

Comment:
There isn't sufficient documentation here. I think we should drop `glean/percentile` from this PR and it can be added later on a standalone basis. I'm also concerned that there's no obvious indication that this is a JS function.